acquia_lift.admin.wizard.inc

```diff
@@ -1409,8 +1437,50 @@ function _acquia_lift_personalize_campaign_wizard_variations_block(&$form, &$for
 
     acquia_lift_chosenify_element($block['block']['bid'], array('acquia-lift-chosen-select-half'));
 
-    // Hide the remove button at this point.
-    $block['remove']['#access'] = FALSE;
+    // Hide the remove button if the options is currently targeted in a running
+    // campaign.
+    $option_id = isset($block['option_id']['#value']) ? $block['option_id']['#value'] : '';
+    if (!empty($option_id) && !$option_delete_allowed && acquia_lift_target_option_targeted($agent_data->machine_name, $option_id)) {
```

Comment:
Minor point but can we rename `$option_delete_allowed` to `$targeted_option_delete_allowed`? Otherwise the logic looks funny as there are scenarios where `$option_delete_allowed` is false but you still allow deleting the option.